lib/editor/tinymce/plugins/kalturamedia/tinymce/js/ltipopuplib.js

```diff
@@ -46,6 +46,7 @@ function insertMedia() {
 
     var url = removeProtocolFromUrl(kafuri);
     url = sourceUrl.replace(kafuri, url);
+    url = removeProtocolFromUrl(url);
```

Comment:
The fact that you're calling this function twice is a bit weird and it makes this whole flow not very clear.
I'd suggest to, if possible, refactor this function.